python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -128,67 +155,125 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
+              value:
+                actor.state === 0 ? null : ActorStateHumanNames[actor.state],
+              priority: "primary",
             },
             {
               label: "Resources",
               value:
-                Object.entries(actor.usedResources).length > 0 &&
-                Object.entries(actor.usedResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.usedResources).length > 0
+                  ? Object.entries(actor.usedResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : null,
+              priority: "primary",
             },
             {
               label: "Pending",
               value: actor.taskQueueLength.toLocaleString(),
+              priority: "primary",
             },
             {
               label: "Executed",
               value: actor.numExecutedTasks.toLocaleString(),
+              priority: "primary",
             },
             {
               label: "NumObjectIdsInScope",
               value: actor.numObjectIdsInScope.toLocaleString(),
+              priority: "secondary",
             },
             {
               label: "NumLocalObjects",
               value: actor.numLocalObjects.toLocaleString(),
+              priority: "secondary",
             },
             {
               label: "UsedLocalObjectMemory",
               value: actor.usedObjectStoreMemory.toLocaleString(),
+              priority: "secondary",
             },
-            // {
-            //   label: "Task",
-            //   value: actor.currentTaskFuncDesc.join(".")
-            // }
           ]
         : [
             {
               label: "ID",
               value: actor.actorId,
+              priority: "primary",
             },
             {
               label: "Required resources",
               value:
-                Object.entries(actor.requiredResources).length > 0 &&
-                Object.entries(actor.requiredResources)
-                  .sort((a, b) => a[0].localeCompare(b[0]))
-                  .map(([key, value]) => `${value.toLocaleString()} ${key}`)
-                  .join(", "),
+                Object.entries(actor.requiredResources).length > 0
+                  ? Object.entries(actor.requiredResources)
+                      .sort((a, b) => a[0].localeCompare(b[0]))
+                      .map(([key, value]) => `${value.toLocaleString()} ${key}`)
+                      .join(", ")
+                  : null,
+              priority: "primary",
             },
           ];
 
+    // Apply transformation to add styling for information field
+    const transforms: Record<string, (info: ActorInformation) => ReactNode> = {
```

Comment:
Can we add an explicit `| undefined` to the return type here since we're using that behavior below?